### DISCUSSION TITLE
HM-ES-TX-WM firmware update 2.5 fails

### DISCUSSION THREAD

**Discussion by autumntea** [OpenCCU/OpenCCU#2484] [🇺🇸 Q&A] [answered]
### Describe the issue you are experiencing

When trying to run an update of the HM-ES-TX-WM meter from the original firmware v1.2 to the latest v2.5 (as downloaded from EQ3) Error 104 appears (might be no valid add-on archive, corrupt or you are trying to install the wrong hardware version).

I also checked with earlier RaspberryMatic versions (e.g. 3.63.8.20220330) and new downloads of the add-on with the same result.

### Describe the behavior you expected

Normal loading of add-on into Raspberrymatic.

### Steps to reproduce the issue

1. Download add-on from EQ3 (https://www.eq-3.de/service/downloads.html)
2. In Raspberrymatic go to Setting > Control panel > Additional software > Install/update additional software
3. Load downloaded tgz file and try to run installation
...

### What is the version this bug report is based on?

3.71.12.20231020

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
no
```

**Comment by jens-maus** [comment, marked as answer]
I just tested it myself, the `HM-ES-TX-WM_update_V2_5_191209.tgz` firmware update package can be perfectly uploaded in the WebUI and is also recognized accoringly. See here:

<img width="1080" alt="Bildschirmfoto 2023-10-21 um 16 42 11" src="https://github.com/jens-maus/RaspberryMatic/assets/5887769/c63ad3c5-14c3-466f-a7ef-84bab01c2905">

So this must be an issue in your own installation/setup. So please make sure you did **NOT** unarchive the `tgz` archive or that your browser is doing that automatically and you are falsely trying to upload the unarchived tar. And if the error is still there please provide more information (screenshot, etc.).

**Reply by autumntea** [reply]
Thank you for the very prompt reply. And sorry for bothering, I was too quick with my post. After restarting everything, it now worked!
